Source/DafnyRuntime/DafnyRuntimeJava/src/main/java/dafny/DafnySequence.java

```diff
@@ -295,9 +375,25 @@ public String verbatimString(){
     public HashSet<T> UniqueElements() {
```

Comment:
Drive-by nit: probably should be `uniqueElements()`?

Reply:
It's uppercase because the name is hardcoded in the `Compiler` base class (`dafny.Helpers.Quantifier()` is the same way), which of course follows C# naming sensibilities.  (I suppose we could also offer a lowercase version for native code.)

Reply:
Okay that's definitely a deeper issue than I'd want to fix now then. :)